README.md

```diff
@@ -1,11 +1,14 @@
 # Metal
 
-A programming language. We'll add a catchy slogan soon, pinky swear
+Prototype compiled language built in Rust with LLVM. We'll add a catchy slogan later.
+
+## Contributing
+
+Currently, Metal is unlicensed and solely owned by Vincent and Lena the maintainers. Metal will eventually open up and
```

Comment:
> owned by <us>
https://choosealicense.com/no-permission/

So say that i believe we'll at the very least need a LICENSE file that says "Copyright <us>. All rights reserved"